src/rpc/rawtransaction.cpp

```diff
@@ -1470,6 +1470,216 @@ UniValue sendrawtransaction(const UniValue &params, bool fHelp)
+            "       ]\n"
+            "    }\n"
+            "}\n"
+            "\nExamples:\n"
+            "\nCreate a transaction\n" +
+            HelpExampleCli("createrawtransaction",
+                "\"[{\\\"txid\\\" : \\\"mytxid\\\",\\\"vout\\\":0}]\" \"{\\\"myaddress\\\":0.01}\"") +
+            "Sign the transaction, and get back the hex\n" + HelpExampleCli("signrawtransaction", "\"myhex\"") +
+            "\nSend the transaction (signed hex)\n" + HelpExampleCli("sendrawtransaction", "\"signedhex\"") +
+            "\nAs a json rpc call\n" + HelpExampleRpc("validaterawtransaction", "\"signedhex\""));
+    }
+
+    RPCTypeCheck(params, boost::assign::list_of(UniValue::VSTR)(UniValue::VBOOL)(UniValue::VSTR));
+
+    // parse hex string from parameter
+    CTransaction tx;
+    if (!DecodeHexTx(tx, params[0].get_str()))
+        throw JSONRPCError(RPC_DESERIALIZATION_ERROR, "TX decode failed");
+    CTransactionRef ptx(MakeTransactionRef(std::move(tx)));
+    const uint256 &hashTx = ptx->GetHash();
+
+    bool fOverrideFees = false;
+    TransactionClass txClass = TransactionClass::DEFAULT;
+
+    // 2nd parameter allows high fees
+    if (params.size() > 1)
+    {
+        if (params[1].isBool())
+        {
+            fOverrideFees = params[1].get_bool();
+        }
+        else if (params[1].isStr())
+        {
+            std::string maybeOverride = params[1].get_str();
+            if (maybeOverride == "allowhighfees")
+            {
+                fOverrideFees = true;
+            }
+            else if (maybeOverride == "allowhighfees")
+            {
+                fOverrideFees = false;
+            }
+            else
+            {
+                throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid allowhighfees value");
+            }
+        }
+        else
+        {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid allowhighfees value");
+        }
+    }
+    // 3rd parameter must be the transaction class
+    if (params.size() > 2)
+    {
+        txClass = ParseTransactionClass(params[2].get_str());
+        if (txClass == TransactionClass::INVALID)
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid transaction class");
+    }
+
+    CCoinsViewCache &view = *pcoinsTip;
+    bool fHaveChain = false;
+    {
+        for (size_t i = 0; !fHaveChain && i < tx.vout.size(); i++)
+        {
+            CoinAccessor existingCoin(view, COutPoint(hashTx, i));
+            fHaveChain = !existingCoin->IsSpent();
+        }
+    }
+    UniValue result(UniValue::VOBJ);
+    bool fHaveMempool = mempool.exists(hashTx);
+    CValidationDebugger debugger;
+    if (!fHaveMempool && !fHaveChain)
+    {
+        CValidationState state;
+        bool fMissingInputs;
+        std::vector<COutPoint> vCoinsToUncache;
+        bool isRespend = false;
+        bool missingInputs = false;
+        ParallelAcceptToMemoryPool(txHandlerSnap, mempool, state, std::move(ptx), false, &fMissingInputs, false,
```

Comment:
missingInputs seems to be an unused variable...

Reply:
It was accidentally added when i changed the call from AcceptToMemoryPool to ParallelAcceptToMemoryPool. removed.